### ISSUE TITLE
please release DBD-MariaDB on CPAN

### ISSUE THREAD

**Issue by glasswalk3r** [gooddata/DBD-MariaDB#80]
Although there are some tests for DBD-MariaDB at the [Test Matrix](http://matrix.cpantesters.org/?dist=DBD-MariaDB#), it seems Maria-DB is not available on CPAN.
That would not only make the distribution easily available for developer, but it would also take advantage of the several smoker testers available around the world.

**Comment by pali** [comment]
Hi! Beta version 0.90_01 is already on CPAN, see:
https://metacpan.org/release/PALI/DBD-MariaDB-0.90_01

Also I sent announcement email, but due to broken perl email infrastructure it was not probably delivered correctly...
https://www.nntp.perl.org/group/perl.dbi.dev/2018/06/msg8174.html

**Comment by pali** [comment]
Version 1.00 was released on CPAN: https://metacpan.org/pod/DBD::MariaDB

**Comment by glasswalk3r** [comment]
Great news @pali , congrats!
I didn't have the time to update here before the release, but you might want to take a look at the reports I sent manually:
http://matrix.cpantesters.org/?dist=DBD-MariaDB%200.90_01
I'll wait a while before updating my local indexes and you should get results automatically.

**Comment by pali** [comment]
@glasswalk3r I saw it and it looks like a memory leak which is OpenBSD specific. I'm afraid, but I do not know OpenBSD memory allocation details to debug it.

**Comment by glasswalk3r** [comment]
Maybe I can help you with that.
How do you usually conduct the debugging of it in Linux?
Are we sure the bug in on the C portion of the module? I once had to deal with a memory leak in pure Perl code [before](https://github.com/theory/fsa-rules/issues/3) and had to not push too hard the tests in order of not getting false results.

**Comment by pali** [comment]
I run that test on CentOS 6, 7, Ubuntu 12.04, 14.04 and Debian 9. All systems have different versions of kernel, perl (interpreter), DBI and probably also glibc (where is malloc). And test passed.

In most cases I debug memory leaks via valgrind tool. Memory leak can be everywhere, in perl part of DBI, in C part of DBI and also in the libmysql/mariadb client. As this problem does not happen on 5 different Linux distributions it is really strange. So there is a small theoretical chance that there is no real memory leak, just OpenBSD malloc implementation has higher consumption of system resources. Which makes it impossible to write such memory leak test in perl based on measurement of process virtual memory size.

I have already started OpenBSD 6.3 in KVM and confirmed that test failed. I have installed valgrind via pkg_add, but seems that valgrind is fully broken on OpenBSD 6.3. It always crash after running some process under it.
